nginx-controller/controller/controller.go

```diff
@@ -1243,6 +1266,11 @@ func (lbc *LoadBalancerController) findMasterForMinion(minion *extensions.Ingres
 		return &extensions.Ingress{}, err
 	}
 
+	// ingresses are sorted by creation time
+	sort.Slice(ings.Items[:], func(i, j int) bool {
```

Comment:
sorting ingresses is unnecessary here, as we will not check for duplicated masters for now

Reply:
Done